daml-lf/spec/value.rst

```diff
@@ -374,21 +379,25 @@ All fields are required and must be non-empty.
    containing record, as declared by the LF record type.  It must be
    supplied in all cases.
 
+(*since version 12*)
+
+As of version 12, the field ``label`` is not used anymore.
```

Comment:
you state above that the field is required. I think the message should be closer to that, otherwise it gets a bit confusing.